### PR TITLE
Clean up variables for terraform part 3

### LINKED ISSUE #105: Clean up terraform part 3
- fixing variables and names
- documenting variables

### PR DESCRIPTION
**Pull request by TessFerrandez**
Closes #105 

- Renames a few of the variables
- Removes unused variables
- Documents the variables used

Also cleans up a small doc issue with flux installation #73 

>**Note:** I keep getting an error "Error: validation: chart.metadata is required" - but I get this with the main branch as well, so can whomever review this walk through all 3 steps before approving to make sure it is not an issue with the scripts

Thank you!

### CODE REVIEW

**Review by tompaana — APPROVED** [review]
LGTM - the first two parts of the trilogy were already awesome, but this last one, especially the twist in the end, really blew my mind. Bravo!

**Comment by sachinkundu** [comment]
Does the naming work @TessFerrandez ?? I think calling the folder flux with the helm chart named flux will create strange problems in helm deployments.

Were you not getting some helm errors??

**Comment by sachinkundu** [comment]
Error: validation: chart.metadata is required" thats it! That's the reason your folder is called flux. I think you can call it anything but flux because this fudges helm chart which are downloaded locally

File: setup.md [hunk 1]
```diff
@@ -260,6 +259,18 @@ This will set up Flux for admin and non-admin workloads and apply the desired st
 
 1. Verify that the Terraform backend values match your storage account in [./3-postprovision/main.tf](3-postprovision/main.tf).
```
Comment (deniscep) [inline setup.md:260]: I don't think there is anything to verify in main.tf?

**Review by deniscep — COMMENTED** [review]
Can confirm Sachin's comment; if the folder is called "flux", you should be seeing "...chart.metadata..." error. Renaming it fixes it. Otherwise PR looks awesome, and runs successfully.

File: setup.md (hunk 1 above)
Comment (TessFerrandez) [reply setup.md:260]: Good catch - will fix this when I re-issue

**Comment by TessFerrandez** [comment]
Abandoning this PR to create a new one on top of Sachins

**Pull request closed without merging**